omego/external.py

```diff
@@ -214,9 +216,12 @@ def run(exe, args, capturestd=False, env=None):
             stderr = errfile.read()
             errfile.close()
 
+        end = time.time()
         if r != 0:
             raise RunException(
-                "Non-zero return code", exe, args, r, stdout, stderr)
+                "Non-zero return code [%s s]" % (end - start), exe, args, r,
```

Comment:
Seems a bit strange to be putting timings in the exception message. How about improve the logging instead: `log.error("Non-zero return code %d [%s s]", r, (end - start))` before the raise?